packages/components/src/dropdown-menu-v2/styles.ts

```diff
@@ -25,12 +25,14 @@ const ITEM_PADDING_BLOCK = space( 2 );
 const ITEM_PADDING_INLINE = space( 3 );
 
 // TODO:
-// - those values are different from saved variables?
-// - should bring this into the config, and make themeable
-// - border color and divider color are different?
-const DEFAULT_BORDER_COLOR = COLORS.gray[ 300 ];
-const DIVIDER_COLOR = COLORS.gray[ 200 ];
-const TOOLBAR_VARIANT_BORDER_COLOR = COLORS.gray[ '900' ];
+// - border color and divider color are different from COLORS.theme variables
+// - lighter text color is not defined in COLORS.theme, should it be?
+// - lighter background color is not defined in COLORS.theme, should it be?
```

Comment:
Maybe these two questions are trivial enough that I can add those variables directly in the PR, if folks agree.

Reply:
I have a feeling we shouldn't be doing too much abstracting or "semanticizing" right now, and rather wait until the design folks are ready with a new token system. I'm assuming this [color system in the Figma](https://www.figma.com/design/804HN2REV2iap2ytjRQ055/Core-System-Library?node-id=2272-16641) is still WIP, but in any case the token system there is very different from the one we currently have in `@wordpress/components`.

Reply:
It's definitely true that the priority is to make such progress directly on the theme effort, which will then represent the source of truth for those design tokens.

I thought we could have a temporary internal set of variables to help with consistency and with the refactor to the new design tokens, but I'm also completely fine leaving the code as-is.